Software/src/LightpackApplication.cpp

```diff
@@ -757,7 +757,7 @@ void LightpackApplication::initGrabManager()
 #endif
 
 	// Connections to signals which will be connected to ILedDevice
-	if (!m_noGui)
+	if (!m_noGui && m_settingsWindow)
```

Comment:
Checking m_noGui is sufficient, or is there a case where `!m_noGui && !m_settingsWindow`?

Reply:
now either combination should be enough, I just put both for extra..
there are places where neither are checked but I didn't trace every path, it seems to run as is though
what broke it is the addition of soundviz on linux (and mac)

Reply:
I thought so far the code was consistently checking m_noGui (except since the addition of soundviz-linux/mac), but fine, it doesn't really matter